requirements.txt

```diff
@@ -3,3 +3,5 @@ numpy>=1.11.3
 pyomo>=5.5.0
 tsam>=0.9.9
 sphinx_rtd_theme>=0.3.1
+matplotlib
```

Comment:
Could we quickly discuss with Dil if we want to have matplotlib in the requirements here? I remember there was some trouble with matplotlib when running FINE models on the cluster.